frontend/app/components/depobio/depobio.component.css

```
@@ -0,0 +1,139 @@
+
+.container-fluid {
+    padding: 2rem 0;
+    background-color: #f8f9fa;
+    min-height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.folder-container {
+    max-width: 60rem;
+    margin: 0 auto;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+    border: none;
+    border-radius: 8px;
+    position: relative;
+}
+
+.card-content {
+    padding: 2rem;
+}
+
+.card-title {
+    color: #2c3e50;
+    font-size: 2.5rem;
+    margin-bottom: 1.5rem;
+    font-weight: 600;
+}
+
+.card-subtitle {
+    color: #6c757d;
+    font-size: 1.2rem;
+    margin-bottom: 2rem;
+    font-weight: 400;
+}
+
+.form-control {
+    width: 100%;
+    max-width: 400px;
+    margin: 0 auto;
+    padding: 0.75rem;
+    border: 2px solid #e9ecef;
+    border-radius: 6px;
+    transition: border-color 0.3s ease;
+}
+
+.form-control:focus {
+    border-color: #007bff;
+    box-shadow: 0 0 0 0.2rem rgba(0, 123, 255, 0.25);
+    outline: none;
+}
+
+.text-center {
+    text-align: center;
+}
+
+
+.search-container {
+    position: relative;
+    max-width: 400px;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+}
+
+.search-button {
+    background-color: #007bff;
+    border: none;
+    border-radius: 6px;
+    padding: 0.75rem 1.5rem;
+    margin-left: 0.5rem;
+    color: white;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+}
+
+.search-button:hover {
+    background-color: #0056b3;
+}
+
+.search-button:disabled {
+    background-color: #cccccc;
+    cursor: not-allowed;
+}
+
+.search-button i {
+    font-size: 1.1rem;
+}
+
+.result-message {
+    margin-top: 1.5rem;
+}
+
+.alert {
+    padding: 1rem;
+    border-radius: 6px;
+    margin: 0 auto;
+    max-width: 400px;
+}
+
+.alert-success {
+    background-color: #d4edda;
+    border-color: #c3e6cb;
+    color: #155724;
+}
+
+.alert-danger {
+    background-color: #f8d7da;
+    border-color: #f5c6cb;
+    color: #721c24;
+}
+.loader {
```

```suggestion
}

.loader {
```

Juste un saut de ligne manquant.